src/python/WMCore/DataStructs/LumiList.py

```diff
@@ -61,7 +64,7 @@ def __init__(self, filename=None, lumis=None, runsAndLumis=None, runs=None, comp
                 self.compactList = json.load(jsonFile)
         elif url:
             self.url = url
-            with closing(urllib2.urlopen(url)) as jsonFile:
+            with closing(urllib.request.urlopen(url)) as jsonFile:
```

Comment:
This is used extensively in dmwm/WMCore. In case of error, it may just be easier to have everything fail where it should instead going through our code and check if the error can be recovered and fail otherwise.

Reply:
Changing L437 of WMConfigCache can have impacts on
https://github.com/cms-PdmV/wmcontrol/blob/1cd4db611d167f73ed4f9ae109ed360a3392eaae/modules/wma.py#L272
for example

Reply:
After some thought, I suggest not to change this line. The behavior is not going to change from  py2 `urllib2` to py3 `urllib`.